source/lib/src/rocm/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# required cmake version
+cmake_minimum_required(VERSION 3.5)
+# project name
+project(deepmd_op_rocm)
+set(CMAKE_LINK_WHAT_YOU_USE TRUE)
+
+#find package ROCM
+list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/../../../cmake/)
```

Comment:
`CMAKE_SOURCE_DIR` should be the top level directory of the CMake source tree. It seems that this command would not set the correct directory to the findROCM.